packages/app/src/contacts/helpers/is-contact-matching-phone-number/is-contact-matching-phone-number.ts

```diff
@@ -3,21 +3,15 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import { removeDecoratorsFromPhoneNumber } from "App/__deprecated__/renderer/models/utils/remove-decorators-from-phone-number"
 import { Props } from "App/contacts/helpers/is-contact-matching-phone-number/is-contact-matching-phone-number.type"
 
-export const isContactMatchingPhoneNumber = (
-  { primaryPhoneNumber = "", secondaryPhoneNumber = "" }: Props,
-  phoneNumber: string
+export const isContactMatchingPhoneNumberId = (
+  { primaryPhoneNumberId = "", secondaryPhoneNumberId = "" }: Props,
+  phoneNumberId: string
 ): boolean => {
-  if (phoneNumber === "") {
+  if (phoneNumberId === "") {
```

Comment:
Is this if necessary? The check after return statement in this case will also return false.